tests/e2e/specs/out-of-game/snackbar.spec.js

```diff
@@ -0,0 +1,47 @@
+import { myUser } from '../../fixtures/userFixtures';
+import en from '../../../../src/translations/en.json';
+
+describe('Lobby - Page Content of ranked game', () => {
+  beforeEach(() => {
+    setup();
+  });
+
+  it('Changes games type. snackbar is visible', () => {
+    //switch to ranked
+    clickGameModeSwitch();
+    cy.get('[data-cy=edit-snackbar]').should('exist');
+    
+  });
+
+  it('Changes games type. check appropriate text', () => {
+    clickGameModeSwitch();
+    cy.get('[data-cy=edit-snackbar]').contains(`${en.lobby.rankedChangedAlert} ${en.global.ranked}`);
+    clickGameModeSwitch();
+    cy.get('[data-cy=edit-snackbar]').contains(`${en.lobby.rankedChangedAlert} ${en.global.casual}`);
+  });
+
+  it('Changes games type. close snackbar', () => {    
+    clickGameModeSwitch();
+    cy.get('[data-cy=close-snackbar]').click();
+    cy.get('[data-cy=edit-snackbar]').should('not.exist');
+  });

```

Comment:
Rather than making each of these a separate test case, we can combine them into the existing lobby test case for switching game modes as additional assertions in a single test